sdks/python/apache_beam/io/textio.py

```diff
@@ -21,7 +21,7 @@
 from __future__ import absolute_import
 import logging
 
-from apache_beam import coders
+from apache_beam.coders import coders
```

Comment:
`coders/__init__.py` changed because of the `__all__` in `coders/coders.py`

This is only needed because it uses the no-public ToString Coder. `beam.coders.Coder` is still recommended.